### ISSUE TITLE
Problems installing URD

### ISSUE THREAD

**Issue by Thralni** [farrellja/URD#49]
Hey,

I have issues installing URD. The installer script is outdated with Bioconductor installations  for R 3.5 and up, as the commands have changed, and so I'm installing manually. All dependencies appear to be installing fine, but I cannot get URD to install.

After devtools announce sit will start to build URD...
`-  building 'URD_1.1.0.tar.gz'`

...First I get a whole bunch of warnings that follow the same basic pattern:
`Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
     storing paths of more than 100 bytes is not portable:
     'URD/Analyses/QuickStart/URD-QuickStart-AxialMesoderm_files/figure-markdown_github/unnamed-chunk-10-1.png'`

Then, this happens:
`Installing package into ‘C:/Users/bjv2113/Documents/R/win-library/3.6’
(as ‘lib’ is unspecified)
* installing *source* package 'URD' ...
** using staged installation
** R
Error : (converted from warning) unable to re-encode 'NMFDoublets.R' line 297
ERROR: unable to collate and parse R files for package 'URD'
* removing 'C:/Users/bjv2113/Documents/R/win-library/3.6/URD'
Error: Failed to install 'URD' from GitHub:
  (converted from warning) installation of package ‘C:/Users/bjv2113/AppData/Local/Temp/RtmpOacWPd/file4ae07e397ec1/URD_1.1.0.tar.gz’ had non-zero exit status`

Any help would be appreciated. URD seems like an interesting package and I'd love to give it a go!

**Comment by farrellja** [comment]
Hi @Thralni 

I'm guessing it's because I had a Greek symbol in a plotting title on that line (297). I think it's something weird with Unicode not correcting getting parsed sometimes. I've changed it in the **debug** branch. Try installing 1.1.0.9005 from the **debug** branch to see whether this is fixed:
```devtools::install_github('farrellja/URD@debug'```

Also, thanks for the heads up about the installation script being broken for R 3.5; I haven't upgraded yet. I'll try to update the installation script.

**Comment by Thralni** [comment]
Thanks @farrellja ! Now it installed without any issues!

Off-topic perhaps, but since I've got your attention... ;) There is one thing currently stopping me from using URD to its fullest potential, and that's the integration with Seurat. Seurat v3 overhauled the Seurat object's structure, so that the current seuratToURD command is outdated. I'm working on redoing some of the low level analyses of Seurat v3 in v2, so that I can then import that to URD, but it'd be great if URD were compatible with Seurat v3's new format. Are there any plans to make that happen?

**Comment by farrellja** [comment]
Hey @Thralni 

I've only just upgraded to Seurat v3 (I don't like to change software while I have active projects... but started some new ones recently in v3). I do plan to incorporate it soon. In the meantime, another user has written a script for doing that import that's posted in Issue #29 , which you could try using. Let me know how it goes, and hopefully I will have time to test one out soon.

Jeff

**Comment by Thralni** [comment]
Hi @farrellja,

Thanks for the comment! The script worked beautifully and I was able to start using URD with my Seurat v3 analysis. However, I did notice that URD, actually, is not unbiased in its approach to pseudotime? One has to define root cells and tip cells, suggesting that I'm biasing the analysis with my ideas of how the data should look like in pseudotime. My data is whole lung epithelium at E14.5, so not an integrated dataset from multiple time points as the tutorial uses.

Benjamin
